gradle-license-plugin/src/test/groovy/com/jaredsburrows/license/LicensePluginAndroidSpec.groovy

```diff
@@ -1,16 +1,13 @@
 package com.jaredsburrows.license
 
-import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
-import static test.TestUtils.assertHtml
```

Comment:
Undo this?

Reply:
Android studio did this for me via optimise imports. Fixed!